dali/pipeline/operator/sequence_operator.h

```diff
@@ -151,9 +158,8 @@ class SequenceOperator : public Operator<Backend>, protected SampleBroadcasting<
   }
 
   bool IsExpandable() const {
-    auto expand_like_idx = GetReferenceInputIdx();
-    assert(expand_like_idx == -1 || GetInputExpandDesc(expand_like_idx).NumDimsToExpand() > 0);
-    return expand_like_idx >= 0;
+    assert(expand_like_ == nullptr || expand_like_->NumDimsToExpand() > 0);
```

Comment:
can you explain the reasoning behind this assert?

Reply:
It stipulates that if you set some input as referential for expansion it must be sequence-like. Just to avoid repeated code of the form

1. GetReferentialExpandDesc();
2. Make sure it is an actual sequence, i.e. there are some dims to expand.